### PR TITLE
improve surge styling, hide warning if not easy

### PR DESCRIPTION
**Pull request by toddmedema**
Was reviewing playtest feedback & did a quick fix for surge warning (only showing it on easy), also added the red background to make surges more visible https://github.com/ExpeditionRPG/expedition-app/issues/342:

screen shot 2017-06-13 at 10 06 15 am

### CODE REVIEW

File: app/cardtemplates/combat/Actions.tsx [hunk 1]
```diff
@@ -81,6 +81,8 @@ function generateCombatAttack(node: ParserNode, settings: SettingsType, elapsedM
   // Scale according to multipliers, then round to nearest whole number and cap at 10 damage/round
   damage = Math.min(10, Math.round(damage * combat.damageMultiplier * playerMultiplier));
 
+console.log(node.ctx.templates.combat);
```
Comment (smartin015) [inline app/cardtemplates/combat/Actions.tsx]: Remove

File: app/components/base/Card.tsx [hunk 2]
```diff
@@ -66,7 +67,7 @@ export default class ExpeditionCard extends React.Component<ExpeditionCardProps,
     }
     // TODO: Spacer ios-only as first child of card style
     return (
-      <div className={'base_card' + ((this.props.dark) ? ' dark' : '')}>
+      <div className={'base_card' + ((this.props.dark) ? ' dark' : '')} style={this.props.background ? {background: this.props.background} : null}>
```
Comment (smartin015) [inline app/components/base/Card.tsx]: The title and button are kind of a dirty brown color currently... could we get a deeper red to better match the background? I wonder if we really need to be turning the `dark` prop into a `theme` prop or some such.

**Comment by toddmedema** [comment]
@smartin015 updated appearance and moved to themes:
screen shot 2017-06-15 at 8 35 27 am

File: app/components/base/Card.test.tsx [hunk 3]
```diff
@@ -12,9 +12,9 @@ test('inner content visible', function() {
 });
 
 test('dark theme applied when dark attribute set', function() {
-  f.setAttribute('dark', true);
+  f.setAttribute("DARK", true);
```
Comment (smartin015) [inline app/components/base/Card.test.tsx]: Technically, `f.setAttribute("theme", "DARK")`

File: app/components/base/Card.tsx [hunk 4]
```diff
@@ -7,14 +7,16 @@ import MoreVertIcon from 'material-ui/svg-icons/navigation/more-vert'
...
+import {CardThemeType} from '../../reducers/StateTypes'
 import {getDevicePlatform} from '../../Globals'
 
 declare var window:any;
 
 
 // If onMenuSelect or onReturn is not set, default dispatch behavior is used.
 interface ExpeditionCardProps extends React.Props<any> {
-  dark?: boolean;
+  background?: string;
```
Comment (smartin015) [inline app/components/base/Card.tsx]: Is `background` still necessary? If not, remove pls.

**Pull request merged**